scripts/create-aks-cluster.sh

```diff
@@ -42,8 +42,11 @@ main() {
     timeout --foreground 1200 bash -c register_feature
     echo "Creating an AKS cluster '${CLUSTER_NAME}'"
     LOCATION="$(get_random_region)"
-    # get the latest patch version of 1.23
-    KUBERNETES_VERSION="$(az aks get-versions --location "${LOCATION}" --query 'orchestrators[*].orchestratorVersion' -otsv | grep '1.23' | tail -1)"
+    # get the latest patch version of 1.22
+    # pinning to 1.22 with upgrade to 1.23 because windows agent pool is not supported for k8s version above 1.24.0
+    # ERROR: (AgentPoolK8sVersionNotSupported) windows agentpool is not supported for k8s version above 1.24.0
+    # TODO(aramase): move this to 1.23 or 1.24 after the issue is fixed
+    KUBERNETES_VERSION="$(az aks get-versions --location "${LOCATION}" --query 'orchestrators[*].orchestratorVersion' -otsv | grep '1.22' | tail -1)"
```

Comment:
Pinning to `1.22` so tests can successfully upgrade to `1.23` and run the tests

Reply:
can we create an issue for this for tracking?